Sources/Router/router.swift

```diff
@@ -0,0 +1,85 @@
+import Foundation
+import Requests
+import Responses
+import Values
+
+public class Router {
+    private var data: DataStorage
+    private var routes: [String: [String]]
+    private var errorMessage: String = "<p> URL does not exist </p>"
+
+    private func routeSetUp() {
+        routes["/"] = ["GET", "POST", "PUT"]
+        routes["/form"] = ["GET", "POST", "PUT"]
```

Comment:
We chatted about this at the IPM - you can use an enum for "GET", "POST", and "PUT"